### ISSUE TITLE
Is Powershell Support Exists

### ISSUE THREAD

**Issue by guneysus** [nickvdyck/webtty#138]
Is it possible to running directly powershell?

Now I can jump powershell by typing `powershell` manually.

**Comment by nickvdyck** [comment]
Yes, this is definitely possible albeit not really obvious and discoverable at the moment. Both things I would like to improve in a coming version.

But you can execute the command like this:
```powershell
webtty -- powershell
```

Then it will launch Powershell by default. On Unix, you have an API call to get the default configured shell, you don't have this on windows that's why I decided to pin it to `cmd`. Anyways the cli works as follows `webtty <cli args> -- <command> <command args>`.

Thanks for your question and bringing this to my attention. I'm going to keep this open as a reminder to improve my docs 😄
